filters/quantitative_ques/README.md

```diff
@@ -13,12 +13,10 @@ default model and datasets were used : `mrm8488/bert-tiny-5-finetuned-squadv2`,
 | Quant | 230| 70.86|
 |Non Quant|1884| 59.02|
 
-Results hint towards the fact that non quantitative questions are more challenging hence measuring performance along this split is important  <br>
 
 ## Why is measuring performance on this split important?
-This filter can be used to sepearte out quantiative questions, it helps to analyse models performance on questions which require numerical understanding.It is also usefull to study possible biases in question generation.
+This filter can be used to sepearte out quantiative questions, it helps to analyse models performance on questions which require numerical understanding.It is also usefull to study possible biases in question generation. <br>Evaluation Results hint towards the fact that non quantitative questions are more challenging hence measuring performance along this split is important  <br>
```

Comment:
minor, typos: separate, useful

Reply:
Thanks for pointing it out , will fix it